system/jhtree/jhtree.cpp

```diff
@@ -964,27 +946,6 @@ CKeyIndex::CKeyIndex(unsigned _iD, const char *_name) : name(_name)
     latestGetNodeOffset = 0;
 }
 
-void CKeyIndex::cacheNodes(CNodeCache *cache, offset_t nodePos, bool isTLK)
-{
-    bool first = true;
-    while (nodePos)
-    {
-        Owned<CJHTreeNode> node = loadNode(nodePos);
-        if (node->isLeaf())
-        {
-            if (!isTLK)
-                return;
-        }
-        else if (first)
-        {
-            cacheNodes(cache, node->getFPosAt(0), isTLK);
-            first = false;
-        }
-        cache->preload(node, iD, nodePos, NULL);
-        nodePos = node->getRightSib();
-    }
-}
-
 void CKeyIndex::init(KeyHdr &hdr, bool isTLK, bool allowPreload)
```

Comment:
is allowPreload parameter now redundant?

Reply:
yes.  I'll go through and remove - but it is quite substantial